README.md

```diff
@@ -1,12 +1,13 @@
 # Distances.jl
 
-[![Build Status](https://travis-ci.org/JuliaStats/Distances.jl.svg?branch=master)](https://travis-ci.org/JuliaStats/Distances.jl)
-[![Coverage Status](https://coveralls.io/repos/JuliaStats/Distances.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/JuliaStats/Distances.jl?branch=master)
+[![Build Status](https://github.com/JuliaStats/Distances.jl/workflows/CI/badge.svg)](https://github.com/JuliaStats/Distances.jl/actions?query=workflow%3ACI)
```

Comment:
This would ensure that only results of the master branch are displayed and linked to:
```suggestion
[![Build Status](https://github.com/JuliaStats/Distances.jl/workflows/CI/badge.svg?branch=master)](https://github.com/JuliaStats/Distances.jl/actions?query=workflow%3ACI+branch%3Amaster)
```